### PR TITLE
Event id to string 2

### PR DESCRIPTION
**Pull request by kz0817**
Some monitoring systems such as OpenStack's ceilometer
handle events as UUID. In order to save that kind of
events, hatohol should also handle them as a string.

### CODE REVIEW

File: server/common/Utils.h [hunk 1]
```diff
@@ -258,6 +258,16 @@ class Utils {
...
 
+	/**
+	 * Calculate addition with the number as a string.
+	 *
+	 * @param num0 A number represented by a string.
+	 * @param num1 An another number.
+	 *
+	 * @return A calculated result.
+	 */
+	static uint64_t add(const std::string &num0, const uint64_t num1);
```
Comment (ashie) [inline server/common/Utils.h]: I prefer "sum()" instead of "add()" for it.
The former one includes a meaning "calculation".

Comment (cosmo0920) [inline server/common/Utils.h]: Really?
I think that  `sum()` implies a calculation for same type values and `add()` implies a add calculation for arbitrary type values.

Comment (ashie) [inline server/common/Utils.h]: I see, let's go without the change.

Comment (kz0817) [inline server/common/Utils.h]: I take the ashie-san's suggestion.
In the context this method is used, sum() seems to be natural.
I think 'add()' is typically used as a non-static method such as
<pre>
X obj = x;
obj.add(5);
</pre>

Comment (cosmo0920) [inline server/common/Utils.h]: Hmm, I see.


> I think 'add()' is typically used as a non-static method

Ah, that's right.
If so, I have no objection.

**Comment by ashie** [comment]
Will you correct HapProcessCeilometer later (in another pull request)?

**Comment by kz0817** [comment]
| Will you correct HapProcessCeilometer later (in another pull request)?

Yes, I will.

**Comment by kz0817** [comment]
>> Will you correct HapProcessCeilometer later (in another pull request)?
> Yes, I will.

Wakabayashi-san is now creating a patch for the problem.
He will send a PR soon after this PR is merged.

**Comment by ashie** [comment]
It works fine on my environment.

**Comment by kz0817** [comment]
Thanks
:relaxed:

**Pull request merged**